Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -794,16 +795,12 @@ def addCKFTracks(
     return s
 
 
-ExaTrkXBackend = Enum("ExaTrkXBackend", "Torch Onnx")
-
-
-def addExaTrkX(
+def addExaTrkx(
```

Comment:
I don't think you meant to change the exatrackx code. Can you remove your changes ?